elodie/geolocation.py

```diff
@@ -161,9 +161,10 @@ def reverse_lookup(lat, lon):
 
     try:
         params = {'format': 'json', 'key': key, 'lat': lat, 'lon': lon}
+        headers = {"Accept-Language": "en-US,en;q=0.5"}
```

Comment:
Does this impact the response? If so we should put it in `constants.py` and later make it configurable.